### ISSUE TITLE
Set Permission in docker file

### ISSUE THREAD

**Issue by maxhaedicke** [FriendlyCaptcha/friendly-lite-server#18]
After the Copy of the Public folder, it would be possible to set the correct Permissions like 

 ```
COPY ./public /var/www/html
COPY ./src /var/www/src

#Set Permissions
RUN chmod -R 755 /var/www/html
```

**Comment by amenk** [comment]
What's the issue / advantage?

**Comment by maxhaedicke** [comment]
I did not set any rights and worked as root user. 
The server was started, but it could not read the php file.

So I set the correct permissions in the Dockerfile and now it works.

This line always ensures that the correct permissions are set for the folder or files in the container 

I had this permissions 
```
-rw-r----- 1 max max  359  2. Feb 07:52 puzzle.php
-rw-r----- 1 max max 1182  2. Feb 07:52 siteverify.php
```
there is a read missing :)

It's just a suggestion, if you don't want the customization that's okay with me too. Just close the issue.

**Comment by amenk** [comment]
How did you start the container?

I guess the change would not do any harm - feel free to make a pull request.

PS: The Dockerfile was originally contributed by @R0Wi - do you have comments?

**Comment by R0Wi** [comment]
In terms of file permissions (since we're using a `php8.2-apache`-based image), user `www-data` should always be able to read  files under `/var/www/html` and `/var/www/src` from inside of the running container. To make sure the permissions are always matching the containers environment, I added the proper `--chown` directives in https://github.com/FriendlyCaptcha/friendly-lite-server/pull/19. See also https://docs.docker.com/engine/reference/builder/#copy

**Comment by amenk** [comment]
Fixed in 0.2.1 
thank you all!
